src/dune_rules/coq_rules.ml

```diff
@@ -446,12 +447,13 @@ let setup_coqdep_rule ~sctx ~loc (cctx : _ Context.t) ~source_rule coq_module =
     ]
   in
   let stdout_to = Coq_module.dep_file ~obj_dir:cctx.dir coq_module in
+  let* coqdep = Context.coqdep ~sctx cctx in
   (* Coqdep has to be called in the stanza's directory *)
-  Super_context.add_rule ~loc sctx
+  Super_context.add_rule ~loc sctx ~dir
```

Comment:
This change is unrelated to this PR, right?

Reply:
I've just eta expanded the definition to make it more explicit that's all.